src/app/containers/PodcastExternalLinks/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, string } from 'prop-types';
+import pathOr from 'ramda/src/pathOr';
+import styled from '@emotion/styled';
+import { C_CLOUD_LIGHT, C_EBON } from '@bbc/psammead-styles/colours';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import { getGreatPrimer } from '@bbc/gel-foundations/typography';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+import Link from './Link';
+
+const Wrapper = styled.div`
+  border-top: 1px ${C_CLOUD_LIGHT} solid;
+  border-bottom: 1px ${C_CLOUD_LIGHT} solid;
+  margin: 0;
+  padding: 0 0 1rem 0;
+`;
+
+const Title = styled.h2`
+  ${({ service }) => getSansRegular(service)}
+  ${({ script }) => getGreatPrimer(script)}
+  color: ${C_EBON};
+`;
+
+const PodcastExternalLinks = ({ links }) => {
+  const { translations, service, script, dir } = useContext(ServiceContext);
+  const defaultTranslation = 'This podcast is also available on';
+  const title = pathOr(
+    defaultTranslation,
+    ['media', 'podcastExternalLinks'],
+    translations,
+  );
+
+  if (!links.length) return null;
```

Comment:
Don't need these lookups if going to return null

```suggestion
  if (!links.length) return null;

  const defaultTranslation = 'This podcast is also available on';
  const title = pathOr(
    defaultTranslation,
    ['media', 'podcastExternalLinks'],
    translations,
  );
```